package.json

```diff
@@ -30,9 +31,9 @@
     "ember-cli-babel": "^5.1.7",
     "ember-cli-eslint": "^3.0.2",
     "ember-cli-version-checker": "^1.3.1",
+    "exists-sync": "0.0.4",
```

Comment:
`fs.existsSync` was deprecated along with `fs.exists` a long way back in https://github.com/nodejs/node/commit/7b5ffa46fe4d2868c1662694da06eb55ec744bde, but then `fs.existsSync` was undeprecated in https://github.com/nodejs/node/commit/7b5ffa46fe4d2868c1662694da06eb55ec744bde.